packages/protocol/src/action/client/ClientRead.ts

```diff
@@ -7,4 +7,7 @@
 import { Read } from "#action/request/Read.js";
 import { ClientRequest } from "./ClientRequest.js";
 
-export interface ClientRead extends Read, ClientRequest {}
+export interface ClientRead extends Read, ClientRequest {
+    /** Set to true to skip the automatic data version injection */
+    skipDataVersionInjection?: boolean;
```

Comment:
Maybe call this `readKnowmVersions`?

Reply:
I changed to "includeKnownVersions" (because that we read is already clear.

Reply:
The problem with that is it is referencing how the request is formulated but could be interpreted as how the response is formulated (include known version data in the read).  "Inject" had the same problem

I was suggesting inverting the value so `readKnownVersions = true` would return data from versions that are known.  Mechanically it would then *not* include versions in the request